extension/plugins/org.polarsys.capella.extension.tools/generated/org/polarsys/capella/extension/tools/patterns/edit/icons/generation/PluginPattern.java

```diff
@@ -208,7 +208,7 @@ public String orchestration(PatternContext ctx) throws Exception {
 	protected void method_doGenerate(final StringBuffer stringBuffer, final PatternContext ctx) throws Exception {
 
 		/**
-		 * Copyright (c) 2002-2010 IBM Corporation and others.
+		 * Copyright (c) 2002-2020 IBM Corporation and others.

```

Comment:
I think we should keep the original copyright unchanged!! @tguiu  can confirms that !
If yes, this file should not be added to this PR